### ISSUE TITLE
No module named 'django_redis'

### ISSUE THREAD

**Issue by bard0x** [g3w-suite/g3w-admin#549]
### Checklist

- [ ] I've searched through the existing [issues](https://github.com/g3w-suite/g3w-admin/issues) and this bug has never been reported before

### Subject of the issue

Hi,

I tried to update to the latest version of g3w but in run phase I have this error:

django.core.cache.backends.base.InvalidCacheBackendError: Could not find backend 'django_redis.cache.RedisCache': No module named 'django_redis'

Why this occurs?

Thanks in advance for the reply.

bard0x

### Steps to reproduce

Update g3w-admin & g3w-client & g3w-suite-docker to the latest version pushed on git

### Environment

- g3w-admin: v3.6.0-alpha.0
- g3w-client: v3.8.0-beta.0
- browser: chrome 
- operating system: windows10

**Comment by wlorenzetti** [comment]
@bard0x how to update? Did you create new docker image?

**Comment by bard0x** [comment]
> @bard0x how to update? Did you create new docker image?

Yes, I recloned from git every project (client, admin & suite-docker)

**Comment by wlorenzetti** [comment]
And after did you recreate the *g3wsuite/g3w-suite:dev* image?

**Comment by bard0x** [comment]
> And after did you recreate the _g3wsuite/g3w-suite:dev_ image?

Yes, exactly

**Comment by bard0x** [comment]
> It's very strange inside `requirements.txt` is set
> 
> https://github.com/g3w-suite/g3w-admin/blob/b61ac04d76f8589d2b02039ee70af576e1a0e8be/requirements.txt#L39
> 
> So on new build It could be installed inside the image.

Yes, I find it too on the requirements.txt but don't works :(
